core/tests/test_utils.py

```diff
@@ -1773,7 +1773,8 @@ def setUp(self):
         self.taskqueue_testapp = webtest.TestApp(main_taskqueue.app)
         self.mail_testapp = webtest.TestApp(main_mail.app)
 
-        self.signup_superadmin_user()
+        if getattr(self, 'SIGNUP_DEFAULT_SUPERADMIN_USER', True):
```

Comment:
This may need a comment, I don't understand where it comes from.

Reply:
Done.